rise/static/main.js

```diff
@@ -483,13 +531,17 @@ function revealMode() {
     buttonHelp();
     $('#maintoolbar').addClass('reveal_tagging');
   } else {
+    var current_cell_index = reveal_cell_index(IPython.notebook);
```

Comment:
You custom function is not necessary, I think... you can use something like this:
```javascript
var current_cell_index = IPython.notebook.get_selected_index()
```